tests/formatters/json.js

```diff
@@ -0,0 +1,33 @@
+(function(){
+    "use strict";
+    var Assert = YUITest.Assert;
+
+    YUITest.TestRunner.add(new YUITest.TestCase({
+
+        name: "JSON formatter",
+
+        "File with no problems should say so": function() {
+            var result = { messages: [], stats: [] },
+                actual = CSSLint.getFormatter("json").formatResults(result, "path/to/FILE", {fullPath: "/absolute/path/to/FILE"});
+            Assert.areEqual("{\"messages\":[],\"stats\":[]}", actual);
+        },
+
+        "Should have no output when quiet option is specified and no errors": function() {
```

Comment:
Shouldn't you define errors & warnings in your result object to test if quiet has an effect? 
What happens if I set `quiet: "false"` ?